app/scripts/controllers/network/createBlockTracker.js

```diff
@@ -0,0 +1,15 @@
+const BlockTracker = require('eth-block-tracker')
+
+/**
+ * Creates a block tracker that sends platform events on success and failure
+ */
+module.exports = function createBlockTracker (args, platform) {
+  const blockTracker = new BlockTracker(args)
+  blockTracker.on('latest', () => {
+    platform && platform.sendMessage && platform.sendMessage({ action: 'ethereum-ping-success' })
+  })
+  blockTracker.on('error', () => {
+    platform && platform.sendMessage && platform.sendMessage({ action: 'ethereum-ping-error' })
```

Comment:
Can we use an `if` block here? We might be getting away with too many of these single-line expressions. 😄

Reply:
Similarly for `on('latest', ...)` above

Reply:
But such terse syntax! Fixed :)